core/range/minmax_spec.rb

```diff
@@ -0,0 +1,66 @@
+require_relative '../../spec_helper'
+
+describe 'Range#minmax' do
+  before(:each) do
+    @x = mock('x')
+    @y = mock('y')
+
+    @x.should_receive(:<=>).with(@y).any_number_of_times.and_return(-1)
```

Comment:
A comment like `# x < y` would be helpful to read to understand the intent of those lines faster